subsys/bluetooth/common/Kconfig

```
@@ -6,7 +6,8 @@
 
 config BT_BUF_ACL_TX_SIZE
 	int "Maximum supported ACL size for outgoing data"
-	range 27 251
+	range 27 251 if BT_LL_SW_SPLIT
```

We may need to use 
```suggestion
	range 27 251 if BT_CTLR
```
instead, but I'm not sure. @cvinayak  please advise

---

```suggestion
```

Host will always fragment to HCI LE Read Buffer Size returned ACL max size.

Remove the conditional, I have updated the Controller implementation to use `MIN(CONFIG_BT_BUF_ACL_TX_SIZE, 251U)` here: https://github.com/zephyrproject-rtos/zephyr/pull/40496